lib_xtcp/.cproject

```diff
@@ -1,1059 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Given the large number of changes to the structure of the library, we should generate a new set of project metadata files for xTIMEcomposer.